### PR TITLE
v0.1.1

### PR DESCRIPTION
**Pull request by simbleau**
Minor patch release to fix view-boxes.

### CODE REVIEW

**Review by DJMcNab — APPROVED** [review]
I have not confirmed that the new version number is correct, but it's good to be doing more regular releases.

File: Cargo.toml [hunk 1]
```diff
@@ -36,5 +36,5 @@ serde_repr = "0.1.18"
 wasm-bindgen-test = "0.3.42"
 
 [features]
-# Enables the wgpu feature on vello, which is disabled by default
+default = []
```
Comment (DJMcNab) [inline Cargo.toml:39]: Hmm, how can this be 0.1.1 if by default we no longer enable the wgpu feature?

Comment (simbleau) [reply Cargo.toml:39]: I put this comment on vello_svg also, but

> I don't want to do semver with this- I want to keep parity with Vello.
> In other words, vello_svg 0.1.X pairs with Vello 0.1
> Semver also really only applies after 1.0

**Review by DJMcNab — CHANGES_REQUESTED** [review]
Same as https://github.com/linebender/vello_svg/pull/14#pullrequestreview-2079468534

**Pull request merged**